servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultRequestTracker.java

```diff
@@ -152,26 +150,26 @@ public final int score() {
         if (concurrentCount > 0 && concurrentStamp != Long.MIN_VALUE) {
             // If we have a request concurrent we should consider how long it has been concurrent so that sudden
             // interruptions don't have to wait for timeouts before our scores can be adjusted.
-            currentEWMA = max(currentEWMA, nanoToMillis(currentTimeNanos - concurrentStamp));
+            currentEWMA = Integer.max(currentEWMA, nanoToMillis(currentTimeNanos - concurrentStamp));
         }
 
         // Add penalty for concurrent requests to account for "unaccounted" load.
         // Penalty is the observed latency if known, else an arbitrarily high value which makes entities for which
         // no latency data has yet been received (eg: request sent but not received), un-selectable.
-        final int concurrentPenalty = (int) min(MAX_VALUE,
-                (long) concurrentCount * concurrentRequestPenalty * currentEWMA);
+        final int concurrentPenalty = (int) Long.min(MAX_VALUE,
+                ((long) concurrentCount) * concurrentRequestPenalty * currentEWMA);
```

Comment:
It's known from the h2 spec and our `ReservableRequestConcurrencyControllers` that we can not have more than `Integer.MAX_VALUE` concurrent requests. Consider changing `concurrentCount` to int

Reply:
`concurrentCount` is already an `int` but I'm deliberately widening it to `long` so that the product `concurrentCount * concurrentRequestPenalty * currentEWMA` doesn't overflow `int`

Reply:
This probably still needs overflow protection even for 3 ints, so I'll change it.

Reply:
It turns out it's not trivial to check for overflows in multiplication so I'm considering punting on that since this happens on the hot path. It would require a fairly large intersection of bad news for it to happen, namely that at least two of the three values would need to be approaching Integer.MAX_VALUE.

Reply:
Should be fixed now.